pkg/types/020/types.go

```diff
@@ -51,6 +67,39 @@ func GetResult(r types.Result) (*Result, error) {
 	return result, nil
 }
 
+func copyIPConfig(from *IPConfig) *IPConfig {
+	if from == nil {
+		return nil
+	}
+	return from.Copy()
+}
```

Comment:
I'm not very clear why this function exists.  The `if nil` bit could go in IPConfig.Copy.

Reply:
@bboreham can't do that, becuase .Copy() is a struct method, and if you can't do `<nil>.Copy()` right?

Reply:
I haven't tried it, but I can't currently see why not.  In Go, `nil` can have a type, but I think all the calls to this one would be statically dispatched.